server.go

```diff
@@ -1480,7 +1480,13 @@ func (s *server) Stop() error {
 		// Disconnect from each active peers to ensure that
 		// peerTerminationWatchers signal completion to each peer.
 		for _, peer := range s.Peers() {
-			s.DisconnectPeer(peer.addr.IdentityKey)
+			err := s.DisconnectPeer(peer.IdentityKey())
+			if err != nil {
+				srvrLog.Errorf("could not disconnect peer: %v"+
```

Comment:
`Warnf` instead? Otherwise may trigger some false positives in the post-submit error scanning.

Reply:
I'd consider this the only blocking statement in this review.

Reply:
👍